### DISCUSSION TITLE
Otl.php line 1650 Undefined offset: 777

### DISCUSSION THREAD

**Discussion by kimsonchang** [mpdf/mpdf#1520] [Q&A]
![图片](https://user-images.githubusercontent.com/21004439/135400434-085418bd-aeea-4380-99e8-1be64b0a70d9.png)
![图片](https://user-images.githubusercontent.com/21004439/135400602-9d9248f1-45b6-46d4-8ad6-c525bfd851b4.png)
 <span class="f-10">văn KHẢI</span>
mpdf/mpdf": "^8.0"

**Comment by finwe** [comment]
What is your question?

**Comment by keithyeohh** [comment]
I am facing the same issue, how do you manage to get it works eventually?
